.gitignore

```
@@ -1,3 +1,32 @@
+# Exclude source archives from the spec directory
+# Origin: https://github.com/github/gitignore/blob/master/Global/Archives.gitignore
+SPECS/**/*.7z
+SPECS/**/*.jar
+SPECS/**/*.rar
+SPECS/**/*.zip
+SPECS/**/*.gz
+SPECS/**/*.gzip
+SPECS/**/*.tgz
+SPECS/**/*.bzip
+SPECS/**/*.bzip2
+SPECS/**/*.bz2
+SPECS/**/*.xz
+SPECS/**/*.lzma
+SPECS/**/*.cab
+SPECS/**/*.xar
+SPECS/**/*.iso
+SPECS/**/*.tar
+SPECS/**/*.dmg
+SPECS/**/*.xpi
+SPECS/**/*.gem
+SPECS/**/*.egg
+SPECS/**/*.deb
+SPECS/**/*.rpm
+SPECS/**/*.msi
+SPECS/**/*.msm
+SPECS/**/*.msp
+SPECS/**/*.txz
```

Looks good, let's just put it in alphabetical order:
```suggestion
SPECS/**/*.7z
SPECS/**/*.bz2
SPECS/**/*.bzip
SPECS/**/*.bzip2
SPECS/**/*.cab
SPECS/**/*.deb
SPECS/**/*.dmg
SPECS/**/*.egg
SPECS/**/*.gem
SPECS/**/*.gz
SPECS/**/*.gzip
SPECS/**/*.iso
SPECS/**/*.jar
SPECS/**/*.lzma
SPECS/**/*.msi
SPECS/**/*.msm
SPECS/**/*.msp
SPECS/**/*.rar
SPECS/**/*.rpm
SPECS/**/*.tar
SPECS/**/*.tgz
SPECS/**/*.txz
SPECS/**/*.xar
SPECS/**/*.xpi
SPECS/**/*.xz
```